posts/2023-10-17-23.0.0.10.adoc

```diff
@@ -0,0 +1,161 @@
+4. Start testing!
+
+Here are some highlights of the changes between Java 18 and Java 21:
+
+* 400: link:https://openjdk.java.net/jeps/400[UTF-8 by Default]
+* 408: link:https://openjdk.java.net/jeps/408[Simple Web Server]
+* 413: link:https://openjdk.java.net/jeps/413[Code Snippets in Java API Documentation]
+* 416: link:https://openjdk.java.net/jeps/416[Reimplement Core Reflection with Method Handles]
+* 418: link:https://openjdk.java.net/jeps/418[Internet-Address Resolution SPI]
+* 421: link:https://openjdk.java.net/jeps/421[Deprecate Finalization for Removal]
+* 422: link:https://openjdk.org/jeps/422[Linux/RISC-V Port]
+* 431: link:https://openjdk.org/jeps/431[Sequenced Collections]
+* 439: link:https://openjdk.org/jeps/439[Generational ZGC]
+* 440: link:https://openjdk.org/jeps/440[Record Patterns]
+* 441: link:https://openjdk.org/jeps/441[Pattern Matching for switch]
+* 449: link:https://openjdk.org/jeps/449[Deprecate the Windows 32-bit x86 Port for Removal]
+* 451: link:https://openjdk.org/jeps/451[Prepare to Disallow the Dynamic Loading of Agents]
+* 452: link:https://openjdk.org/jeps/452[Key Encapsulation Mechanism API]
+
+But perhaps the most anticipated one of all is the introduction of virtual threads in Java 21:
+
+* 444: link:https://openjdk.org/jeps/444[Virtual Threads]
+
+Virtual threads were designed to provide higher throughput for running tasks that spend most of their time blocked, like waiting for I/O operations. Will the impact of virtual threads live up to the anticipation?  Find out for yourself by trying them out in your applications that run on the best Java runtime, Open Liberty!
+
+For more information on Java 21, see:
+
+* link:https://jdk.java.net/21/release-notes[Java 21 release notes]
+* link:https://docs.oracle.com/en/java/javase/21/docs/api/index.html[API Javadoc page]
+* link:https://docs.oracle.com/en/java/javase/21/migrate/getting-started.html[migration guide]
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/26535
+// Contact/Reviewer: jjiwooLim
+// // // // // // // // 
+[#feature]
+== featureUtility now verifies feature signatures by default
+
+The `featureUtility` command now verifies feature signatures before installing the feature into the Liberty runtime. 
+
+Previously, the `featureUtilty` tool only verified checksums. While checksums are essential for integrity (showing that the file has not been tampered with), verifying checksums did not ensure the authenticity of downloaded files.
+
+The command now identifies whether the feature originated from the Liberty development team or is a third-party user feature. We've implemented an additional step in the process of verifying feature signatures to check both the authenticity and integrity of features that are downloaded from the Maven Central repository.
```

Comment:
The last sentences here seems to be repeating (summarizing) what's been previously stated.  In other words, the section goes:
We now do authenticity checks
We only did integrity checks (and here's what an integrity check is)
We now do an authenticity check (and here's what that means)
We now do authenticity and integrity checks.

So it feels a bit redundant..... but not really a problem

Reply:
I just spotted a typo in your extract, so I'll fix both now.